README.md

```diff
@@ -13,7 +13,17 @@ workflow by running during a component's `didRender` phase in non-production
 environments. This gives you instant feedback on if your components are
 accessible in any given state.
 
-## Installation
+
+Compatibility
+------------------------------------------------------------------------------
+
+* Ember.js v1.13 or above
```

Comment:
We should change this to 2.18 (since that is all we are testing)

Reply:
fixed